ova/ovf/centos.ovf

```diff
@@ -138,6 +138,7 @@ file.
       <vmw:Config ovf:required="false" vmw:key="firmware" vmw:value="bios"/>
       <vmw:Config ovf:required="false" vmw:key="virtualICH7MPresent" vmw:value="false"/>
       <vmw:ExtraConfig ovf:required="false" vmw:key="nvram" vmw:value="ovf:/file/file2"/>
+      <vmw:ExtraConfig ovf:required="false" vmw:key="disk.enableUUID" vmw:value="1"/>
```

Comment:
I suppose these keys are case-insenstive, but docs/src for this one is `disk.EnableUUID`

Reply:
son of a...  I thought I double checked that.  Will fix!